server/src/main/scala/com/github/saxypandabear/songrequests/websocket/TwitchSocket.scala

```diff
@@ -31,9 +31,10 @@ class TwitchSocket(
     pingFrequencyMs: Long = 60000
 ) extends LazyLogging {
 
-  private val spotifyUriPattern = "^(spotify:track:(\\w|\\d)+)$".r
-  private val pingTask          = new Timer()
-  private var session: Session  = _
+  private val ExpectedRewardTitle = "TwitchSongRequests"
```

Comment:
You know, I'm not all that picky, but I think for constants I should be doing capitalized variable names like what you did here. 🤔 but I didn't. Maybe I'll go through and update those everywhere some other time. I think I have done all caps and snake case

Reply:
I wasn't sure what scala style was so I just referenced [this guide](https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods). I wasn't sure if the codebase overall had a style choice for constants

Reply:
We can come up with one but not required at all for this. I'm fine leaving this, updating other constants, and then defining that in a style guide